<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpritePropertiesWindow</class>
 <widget class="QDialog" name="SpritePropertiesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>157</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="verticalSpacing">
     <number>12</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="projectNameLabel">
      <property name="text">
       <string>Project Name</string>
      </property>
      <property name="buddy">
       <cstring>projectNameLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="projectNameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="spriteSizeLabel_2">
      <property name="text">
       <string>Sprite Size</string>
      </property>
      <property name="buddy">
       <cstring>spriteSizeSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spriteSizeSpinBox">
      <property name="minimum">
       <number>8</number>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="errorMessage">
      <property name="styleSheet">
       <string notr="true">fontcolor: red;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
